### ISSUE TITLE
RDBES: RS_UnitType update of codes

### ISSUE THREAD

**Issue by HenrikK-N** [ices-eg/DIG#163]
Please see attached spreadsheet for the 3 code updates.

[RS_UnitType updates.xlsx](https://github.com/ices-eg/DIG/files/6931296/RS_UnitType.updates.xlsx)

**Comment by cekv** [comment]
@HenrikK-N RMG suggests changing RS_UnitType to SamplingUnit. Specify difference between box, tray and container for the description.

**Comment by HenrikK-N** [comment]
@cekv yes we can change the name of the RS_UnitType to SamplingUnit, though it is very late in the process, this code list have been created more that an year ago. Box, tray and container are codes which would leave flexibility to the observers and data submitters, so they should not be too specific, because they should used across countries. If RMG want the core group to change the descriptions for some of the existing codes, I suggest RMG creating a new issue, so the creation of the 3 new codes are not delayed, because that is a different request, as we already have agreed at the previous meeting we had with the RMG before summer. We have a dead line for this the 19 Aug because of the data call. If you create a new issue I can take it up with the core group, this is different issue than creating the 3 new requested codes, so it is important to keep this task moving for now.
